code/modules/mob/living/basic/boss/thing/thing_objects.dm

```diff
@@ -58,6 +58,15 @@
 		src.duration = duration
 	return ..()
 
+/obj/effect/temp_visual/telegraphing/exclamation/following/Initialize(mapload, duration, obj/following)
```

Comment:
You should probably give this a type definition instead of just defining it by inference

Also if you set `glide_size` to be the same as `following.glide_size` the animation might be smoother